lcls-twincat-motion/Library/POUs/Motion/PMPS/FB_PositionStatePMPS.TcPOU

```diff
@@ -14,6 +14,7 @@ VAR_IN_OUT
 END_VAR
 VAR_INPUT
     stHighBeamThreshold: ST_BeamParams := PMPS_GVL.cstFullBeam;
+    bAutoReset: BOOL := False;
```

Comment:
This isn't a good name for the variable, since it implies that the entire set can be auto-reset. I don't know if this is the best either but it's more clear the intent.
```suggestion
    bBPOKAutoReset: BOOL := False;
```

Reply:
making this specific to beam parameters, makes sense.